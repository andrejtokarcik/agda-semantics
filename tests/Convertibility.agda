module Convertibility where

postulate x : Set

y : Set1
y = x
