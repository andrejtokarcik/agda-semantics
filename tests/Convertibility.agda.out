<k>
    Set ( 0 ) #== Set ( 1 ) ~> Set ( 1 ) == Set ( 0 ) ~> addFunClause ( Fun ( 
      "y" ) , Fun ( "y" ) , Name ( "x" ) ) ~> unifyConf ~> killMgu ~> 
      cleanMetas ~> loadCtx ~> .List{";"} ~> .List{";"} ~> .List{""}
</k>
<mgu>
    subst(.KList)
</mgu>
<ctx-stack>
    ListItem(#map ( .Map ))
</ctx-stack>
<sig>
    Name ( "x" ) |-> Set ( 0 )
    Fun ( "y" ) |-> Set ( 1 )
</sig>
<ctx>
    .Map
</ctx>
<meta>
    .Map
</meta>
<ncat>
    Name ( "y" ) |-> Fun ( "y" )
</ncat>
<fun>
    .Map
</fun>
